okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java

```diff
@@ -108,7 +109,8 @@ public ModuleManager(ModuleStore moduleStore, boolean local) {
         }
       }
       if (check) {
-        String res = DepResolution.checkDependencies(tempList.values(), newList);
+        String res = DepResolution.checkDependencies(tempList.values(), newList,
+            removeIfMissingDep);
         if (!res.isEmpty()) {
```

Comment:
If both `check` and `removeIfMissingDep` are true, the `res` will always be empty, correct?

Reply:
Correct. No errors would be reported, then.